common/config/rush/pnpm-lock.yaml

```diff
@@ -21511,10 +21535,11 @@ packages:
     dev: false
 
   file:projects/storage-file-share.tgz:
-    resolution: {integrity: sha512-XKoapSGfj9adWoSaAEOMkONC7brW8Pez+vfuoU0Viw46LuU2KTH9ZMWkRYDub+3xt331YRTyWuxdwvbKVzk/Kg==, tarball: file:projects/storage-file-share.tgz}
+    resolution: {integrity: sha512-XtfyjlIQ+IAGLFJpUhDlTWQ99jb5Dv0TIm2uJaWIWHGhTg/xAWlA9gnBd97Rv7FwH+YivSGTfnALNOfhittnIg==, tarball: file:projects/storage-file-share.tgz}
     name: '@rush-temp/storage-file-share'
     version: 0.0.0
     dependencies:
+      '@azure/identity': 3.2.4
```

Comment:
weird that this is the only dev dependency that updated

Reply:
You mean under storage? Or just the only place where identity got updated?

Reply:
I believe the weekly updates on pnpm file take care of most other updates.